### ISSUE TITLE
does it work without finetuning directly using pretrained model for detection?

### ISSUE THREAD

**Issue by jinfagang** [hustvl/MIMDet#1]
does it work without finetuning directly using pretrained model for detection?

**Comment by Yuxin-CV** [comment]
The task layer for object detection (i.e., Mask R-CNN & FPN) is not pre-training and therefore randomly initialized, so I don't think it will work if you don't update the parameter of the task layer.

**Comment by jinfagang** [comment]
@Yuxin-CV thanks for your reply. What if frozen MIM backbone and just finetune the task layer and new classifier? Just wonder how good it can be just direcly using pretrained weights without any modification.

**Comment by Yuxin-CV** [comment]
> @Yuxin-CV thanks for your reply. What if frozen MIM backbone and just finetune the task layer and new classifier? Just wonder how good it can be just direcly using pretrained weights without any modification.

We didn't try that, but to my knowledge， the mim represention isn't very good at linear probing in imagenet 1k，so I guess the same thing will happen on COCO if you freeze the pretrained represention.

We will try this in the future.

**Comment by jinfagang** [comment]
Thank u. I am just out of curiosity, since mim is good at general representation from classification to object detection.
